core/src/main/scala/cromwell/core/WorkflowState.scala

```diff
@@ -10,9 +10,9 @@ sealed trait WorkflowState {
 }
 
 object WorkflowState {
-  private lazy val WorkflowState = Seq(WorkflowSubmitted, WorkflowRunning, WorkflowFailed, WorkflowSucceeded, WorkflowAborting, WorkflowAborted)
+  private lazy val WorkflowStateValues = Seq(WorkflowSubmitted, WorkflowRunning, WorkflowFailed, WorkflowSucceeded, WorkflowAborting, WorkflowAborted)
```

Comment:
because at some point I wanted to reference the `WorkflowState` values and scala got confused and thought I was talking about the object.